src/py42/clients/orgs.py

```diff
@@ -1,8 +1,224 @@
+            u"value": bool_to_str(value),
+            u"locked": False,
+        }
+        if not value:  # disable everything but FMC, like the UI does
+            self.endpoint_monitoring_removable_media_enabled = False
+            self.endpoint_monitoring_browser_and_applications_enabled = False
+            self.endpoint_monitoring_cloud_sync_enabled = False
+
+    @property
+    def endpoint_monitoring_removable_media_enabled(self):
+        value = self._t_settings[u"org-securityTools-device-detection-enable"][u"value"]
+        return str_to_bool(value)
+
+    @endpoint_monitoring_removable_media_enabled.setter
+    @bool_required
+    def endpoint_monitoring_removable_media_enabled(self, value):
+        if value:
+            self.endpoint_monitoring_enabled = True
+        self._t_settings[u"org-securityTools-device-detection-enable"] = {
+            u"key": u"org-securityTools-device-detection-enable",
+            u"value": bool_to_str(value),
+            u"locked": False,
+        }
+
+    @property
+    def endpoint_monitoring_cloud_sync_enabled(self):
+        value = self._t_settings[u"org-securityTools-cloud-detection-enable"][u"value"]
+        return str_to_bool(value)
+
+    @endpoint_monitoring_cloud_sync_enabled.setter
+    @bool_required
+    def endpoint_monitoring_cloud_sync_enabled(self, value):
+        if value:
+            self.endpoint_monitoring_enabled = True
+        self._t_settings[u"org-securityTools-cloud-detection-enable"] = {
+            u"key": u"org-securityTools-cloud-detection-enable",
+            u"value": bool_to_str(value),
+            u"locked": False,
+        }
+
+    @property
+    def endpoint_monitoring_browser_and_applications_enabled(self):
+        value = self._t_settings[u"org-securityTools-open-file-detection-enable"][
+            u"value"
+        ]
+        return str_to_bool(value)
+
+    @endpoint_monitoring_browser_and_applications_enabled.setter
+    @bool_required
+    def endpoint_monitoring_browser_and_applications_enabled(self, value):
+        if value:
+            self.endpoint_monitoring_enabled = True
+        self._t_settings[u"org-securityTools-open-file-detection-enable"] = {
+            u"key": u"org-securityTools-open-file-detection-enable",
+            u"value": bool_to_str(value),
+            u"locked": False,
+        }
+
+    @property
+    def endpoint_monitoring_file_metadata_enabled(self):
+        value = self._t_settings[u"device_fileForensics_enabled"][u"value"]
+        return str_to_bool(value)
+
+    @endpoint_monitoring_file_metadata_enabled.setter
+    @bool_required
+    def endpoint_monitoring_file_metadata_enabled(self, value):
+        if value:
+            self.endpoint_monitoring_enabled = True
+        self._t_settings[u"device_fileForensics_enabled"] = {
+            u"key": u"device_fileForensics_enabled",
+            u"value": bool_to_str(value),
+            u"locked": False,
+        }
+
+    def update(self):
```

Comment:
should _org, _settings, and _t_settings be cleared after an update command resolves?